.github/workflows/remove-template.yml

```diff
@@ -96,8 +96,8 @@ jobs:
   fail-removal-comment:
     name: Removal failed
     runs-on: ubuntu-latest
-    needs: [validate-requestor, remove-from-registry]
-    if: ${{ always() && (needs.remove-from-registry.outputs.error != '' || needs.validate-requestor.outputs.error != '') }}
+    needs: [validate-requestor, validate-admin, remove-from-registry]
+    if: ${{ always() && (needs.remove-from-registry.outputs.error != '' || (needs.validate-requestor.outputs.error != '' && needs.validate-admin.outputs.is-admin != 'active')) }}
```

Comment:
@AmyJZhao , it looks like "validate-requestor" should depend on "validate-admin".
Does it make sense to run 2 these checks in parallel?

Reply:
I updated the workflow to do validate-admin (passed) -> removal & validate-admin (failed) -> validate-requestor (passed) -> removal